### PR TITLE
Add visibility to React Native's feature requests issue

### PR DESCRIPTION
**Pull request by florentroques**
Proposal:
Add link to React Native's feature requests issue to readme so that more people see it and vote!!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,6 +11,10 @@ Swiper component for React Native featuring **previews**, **snapping effect**, *
 
 -----
 
+### Do you want a better plugin? [Vote for React Native's feature requests](https://github.com/archriss/react-native-snap-carousel/issues/203)
```
Comment (bd-arc) [inline README.md]: @florentroques Do you mind modifying it to the following? A small emoji and a bit of clarification cannot harm ;-)
```
### :sparkles: Do you want an even better plugin? [Vote for React Native's feature requests](https://github.com/archriss/react-native-snap-carousel/issues/203) to let the Facebook team know what they need to improve!
```

Comment (florentroques) [reply README.md]: sure

**Comment by florentroques** [comment]
done :-) thank you

**Comment by bd-arc** [comment]
Thank **you** ;-)

I know nothing about your other suggestion: is it possible to display a message in the console when users install the plugin via `npm` or `yarn`?

**Comment by florentroques** [comment]
I've suggested this as I thought it was a good idea but don't actually know its feasibility yet ^_^
Looking into it!

**Comment by florentroques** [comment]
I've looked into displaying a message after the plugin has been installed via npm or yarn.

The hook to use is  `postinstall`, apparently it's only possible if the user uses npm not yarn (security issues) (from [https://github.com/yarnpkg/yarn/issues/721#issuecomment-253058272](https://github.com/yarnpkg/yarn/issues/721#issuecomment-253058272))

> By default, npm also allows developers who write these packages to execute other code that’s needed as part of the install process. That creates security issues, though, so Yarn does not have this feature.

-- [Source](https://techcrunch.com/2016/10/11/facebook-partners-with-google-others-to-launch-a-new-javascript-package-manager/)

Here's what it does

screen shot 2017-11-13 at 18 10 33
screen shot 2017-11-13 at 18 10 05

Here's a test repo https://github.com/florentroques/npm-package-test-postinstall

I open a pull request to implement this

**Pull request merged**